core/domain/opportunity_jobs_one_off_test.py

```diff
@@ -356,3 +357,106 @@ def test_regeneration_job_for_deleted_topic_returns_empty_list_output(self):
 
         output = exp_opp_summary_model_regen_job_class.get_output(job_id)
         self.assertEqual(output, [])
+
+
+class SkillOpportunityModelRegenerationJobTest(test_utils.GenericTestBase):
```

Comment:
@nithusha21, Can you please add this job in job_registry file?

Reply:
Done!